app/views/articles/index.html.erb

```diff
@@ -36,6 +36,11 @@
 
     <%# BEGIN Feed menu bar %>
     <div class="articles-list" id="articles-list">
+
+      <% if user_signed_in? && current_user.created_at > 7.days.ago %>
```

Comment:
Reminder that we don't have access to `current_user` in this context. You'd probably need to hide this HTML and show it via `userData()` in the JS if the user qualifies for this condition.

Reply:
Ah, thanks for this. I was misled by the use of `user_signed_in?` in another part of the file thinking that it's okay to use this here.